MediaBrowser.Controller/Library/IUserManager.cs

```diff
@@ -16,11 +16,19 @@ namespace MediaBrowser.Controller.Library
     /// </summary>
     public interface IUserManager
     {
+        // REVIEW: Possibly make this an ICollection
```

Comment:
I think `IEnumerable` is the right class here since enumarating over the users is pretty much the only things that should be possible here.